up-l1/zenoh.adoc

```diff
@@ -138,33 +147,28 @@ UUri {
 }
 ----
 
-It's a local UUri, so it starts with `upl`.
-Also, we leave the second part blank since it doesn't have UAuthority.
-Only ID in UEntity and UResource matters while serializing into link:../basics/uri.adoc#421-local-micro-uri[micro-form Uri].
-Then transform it into lowercase hexadecimal representation.
-The Zenoh key is `upl/0100162e04d20100`.
+It's a local URI, and the short format of the UUri should be `/1/1234/5678` in this case.
+Let's assume the UAuthority is `192.168.1.100`.
```

Comment:
I'm unclear how we can assume that when we have just said UAuthority is not set because it's a local URI. This should probably be clarified.

Reply:
Maybe I should add more description. UAuthority is defined when creating the UPClientZenoh.

Reply:
Done.